database/mysql/variables.tf

```diff
@@ -0,0 +1,518 @@
+  description = "Time range reflecting the periodicity of the data stream"
+  type        = string
+  default     = "1d"
+}
+
+variable "mysql_questions_anomaly_num_windows" {
+  description = "Number of previous periods used to define baseline, must be > 0"
+  type        = number
+  default     = 4
+}
+
+variable "mysql_questions_anomaly_fire_growth_rate_threshold" {
+  description = "Change over historical norm required to fire, should be >= 0"
+  type        = number
+  default     = 0.2
+}
+
+variable "mysql_questions_anomaly_clear_growth_rate_threshold" {
+  description = "Change over historical norm required to clear, should be >= 0"
+  type        = number
+  default     = 0.1
+}
+
+variable "mysql_questions_anomaly_orientation" {
+  description = "Specifies whether detect fires when signal is above, below, or out-of-band (Options:  above, below, out_of_band)"
+  type        = string
+  default     = "above"
+}
+
+# Mysql_replication_lag detectors
+
+variable "mysql_replication_lag_disabled" {
+  description = "Disable all alerting rules for mysql_replication_lag detector"
+  type        = bool
+  default     = null
+}
+
+variable "mysql_replication_lag_disabled_critical" {
+  description = "Disable critical alerting rule for mysql_replication_lag detector"
+  type        = bool
+  default     = null
+}
+
+variable "mysql_replication_lag_disabled_warning" {
+  description = "Disable warning alerting rule for mysql_replication_lag detector"
+  type        = bool
+  default     = null
+}
+
+variable "mysql_replication_lag_notifications" {
+  description = "Notification recipients list for every alerting rules of mysql_replication_lag detector"
+  type        = list
+  default     = []
+}
+
+variable "mysql_replication_lag_notifications_warning" {
+  description = "Notification recipients list for warning alerting rule of mysql_replication_lag detector"
+  type        = list
+  default     = []
+}
+
+variable "mysql_replication_lag_notifications_critical" {
+  description = "Notification recipients list for critical alerting rule of mysql_replication_lag detector"
+  type        = list
+  default     = []
+}
+
+variable "mysql_replication_lag_aggregation_function" {
+  description = "Aggregation function and group by for mysql_replication_lag detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ""
+}
+
+variable "mysql_replication_lag_transformation_function" {
+  description = "Transformation function for mysql_replication_lag detector (i.e. \".mean(over='5m')\"))"
+  type        = string
+  default     = ".min(over='15m')"
+}
+
+variable "mysql_replication_lag_threshold_critical" {
+  description = "Critical threshold for mysql_replication_lag detector"
+  type        = number
+  default     = 1
+}
+
+variable "mysql_replication_lag_threshold_warning" {
+  description = "Warning threshold for mysql_replication_lag detector"
+  type        = number
+  default     = 0.5
```

Comment:
Seems a bit low to me, and not sure that the `Seconds_Behind_Master` field can go under 1 second. Default values in datadog monitors where much higher (100 and 200).

Reply:
ok